README.md

```diff
@@ -7,13 +7,24 @@ Pulumi's framework for Amazon Web Services (AWS) infrastructure.
 
-* `Network` provides a way to create a new network in your account that follows the [AWS Best Practices](
-  https://aws.amazon.com/answers/networking/aws-single-vpc-design/) for doing so.  This includes creating a Virtual
-  Private Cloud (VPC) with public and private subnets, if you so choose, and multi-availability zone configuration.
+The AWS Infrastructure package undergoes constant improvements and additions.  While we will strive to maintain backward compatability here, we will occasionally make breaks here as appropriate if it helps improve the overall quality of this package.
 
-* `Cluster` offers a simple way of creating an Elastic Container Service (ECS) cluster for running containers.
+The AWS Infrastructure package exposes many high level abstractions.  Including, but not limited to:
+
+* ~~`Network`~~. Deprecated.  Use [ec2.VPC](https://github.com/pulumi/pulumi-aws-infra/blob/0b432e320c6929866038507e997d55c8d8f62bc3/nodejs/aws-infra/ec2/vpc.ts#L25) instead.
+
+* ~~`Cluster`~~. Deprecated.  Use [ecs.Cluster](https://github.com/pulumi/pulumi-aws-infra/blob/0b432e320c6929866038507e997d55c8d8f62bc3/nodejs/aws-infra/ecs/cluster.ts#L26) instead.
+
+* [`autoscaling`](https://github.com/pulumi/pulumi-aws-infra/blob/master/nodejs/aws-infra/autoscaling). A module that makes creating auto-scaling launch configurations and launch groups easier.   Note: given an aws-infra `ecs.Cluster`, the easiest way to create an auto-scaling group is to just call [`createAutoScalingGroup`](https://github.com/pulumi/pulumi-aws-infra/blob/0b432e320c6929866038507e997d55c8d8f62bc3/nodejs/aws-infra/ecs/cluster.ts#L78) on it.  This will produce an auto-scaling group already property configured for that cluster.
+
+* [`ec2`](https://github.com/pulumi/pulumi-aws-infra/blob/master/nodejs/aws-infra/ec2).  A module that makes it easier to work with your AWS network, subnets, and security groups.  By default, the resources in the package follow the [AWS Best Practices](
+  https://aws.amazon.com/answers/networking/aws-single-vpc-design/), but can be configured as desired in whatever ways you want.  Most commonly, this package is used to acquire the default Vpc for a region (using [`Vpc.getDefault`](https://github.com/pulumi/pulumi-aws-infra/blob/0b432e320c6929866038507e997d55c8d8f62bc3/nodejs/aws-infra/ec2/vpc.ts#L118).  However, it can also be used to easily create or augment an existing Vpc.
+
+* [`ecs`](https://github.com/pulumi/pulumi-aws-infra/blob/master/nodejs/aws-infra/ecs).  A module that makes it easy to create and configure clusters, tasks and services for running containers. Convenience resources are created to make the common tasks of creating EC2 or Fargate services and tasks much simpler.  
```

Comment:
Should we make things like "clusters", "tasks" and "services" be `Cluster`s, `Task`s and `Service`s with links to docs?